lib/nx_signal.ex

```diff
@@ -224,6 +301,107 @@ defmodule NxSignal do
     output
   end
 
+  defnp as_windowed_reflect_padding(tensor, opts \\ []) do
```

Comment:
Reflect padding might be a good addition upstream to Nx

Reply:
The main difficulty with that is that currently padding will create values that are repeated (through {left, right, inner} padding config tuples).

I can't see how we'd frame this in terms of that API

Reply:
I mean the API that talks to backends, not the public-facing one